src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -191,23 +116,16 @@ internal string GetDebugString()
                 sb.Append(nameof(this.ExtendedSessionIdleTimeoutInSeconds)).Append(": ").Append(this.ExtendedSessionIdleTimeoutInSeconds).Append(", ");
             }
 
-            sb.Append(nameof(this.EventGridTopicEndpoint)).Append(": ").Append(this.EventGridTopicEndpoint).Append(", ");
-            if (!string.IsNullOrEmpty(this.EventGridTopicEndpoint))
+            if (this.Notifications != null)
             {
-                sb.Append(nameof(this.EventGridKeySettingName)).Append(": ").Append(this.EventGridKeySettingName).Append(", ");
-                sb.Append(nameof(this.EventGridPublishRetryCount)).Append(": ").Append(this.EventGridPublishRetryCount).Append(", ");
-                sb.Append(nameof(this.EventGridPublishRetryInterval)).Append(": ").Append(this.EventGridPublishRetryInterval).Append(", ");
-                sb.Append(nameof(this.EventGridPublishRetryHttpStatus)).Append(": ").Append(string.Join(", ", this.EventGridPublishRetryHttpStatus ?? new int[0])).Append(", ");
+                this.Notifications.AddToDebugString(sb);
             }
 
-            if (this.NotificationUrl != null)
+            if (this.Tracing != null)
```

Comment:
This null check might not be necessary since we always default `this.Tracing` above to a new `TracingOptions` object.

Reply:
Removed it.